---
title: "Pedro Baldoni"
image: "avatar.png"
links:
  - label: Google Scholar
    url: "https://scholar.google.com/citations?user=5lHSzXYAAAAJ"
  - label: GitHub
    url: "https://github.com/plbaldoni"
  - label: Email
    url: "mailto:pedrobaldoni@gmail.com"
output:
  postcards::trestles:
    self_contained: false # remotes::install_github("seankross/postcards@self-contained")
---

## About

I am an Assistant Professor in the [Department of Biostatistics and Health Data Science](https://www.publichealth.pitt.edu/biostatistics) at the University of Pittsburgh. Previously, I was a postdoctoral researcher in the laboratory of [Prof. Gordon Smyth](https://www.wehi.edu.au/laboratory/smyth-lab) at the Walter and Eliza Hall Institute of Medical Research. I received my PhD in Biostatistics from the University of North Carolina at Chapel Hill under the supervision of [Dr. Naim Rashid](https://sph.unc.edu/adv_profile/naim-rashid-phd/) and [Dr. Joseph Ibrahim](https://sph.unc.edu/adv_profile/joseph-g-ibrahim-phd).

My research interests focus on developing statistical methods and open-source bioinformatic tools to analyze data from a wide range of high-throughput genomic, transcriptomic, and proteomic technologies. I enjoy creating new statistical methods and writing efficient bioinformatic tools that help researchers interpret their data. I am particularly interested in developing methods and software for the [Bioconductor Project](https://bioconductor.org). One key goal of the methods I develop is to identify molecular features, such as genomic coordinates, genes/transcripts, or proteins, that change in accessibility, expression, or abundance between experimental conditions.

## Projects

* Differential transcript usage with limma and edgeR ![](https://bioconductor.org/shields/years-in-bioc/limma.svg) ![](https://bioconductor.org/shields/years-in-bioc/edgeR.svg)
    + Uncertainty-aware differential transcript usage analysis method with divided counts. The diffSplice function is now a generic function with methods for limma MArrayLM objects and edgeR DGEGLM objects for Bioconductor v3.21 and above.
    + [limma](https://bioconductor.org/packages/limma) and [edgeR](https://bioconductor.org/packages/edgeR)
    + [Workflow](https://plbaldoni.github.io/DTU-code)
    + [Preprint](https://doi.org/10.1101/2025.04.07.647659)

* Differential transcript expression with edgeR v4 ![](https://bioconductor.org/shields/years-in-bioc/edgeR.svg)
    + Improved differential transcript expression pipeline with [Salmon](https://github.com/COMBINE-lab/salmon)'s Gibbs sampling and the new bias-corrected quasi-likelihood method with adjusted deviances for small counts from edgeR v4.
    + [R/Bioconductor](https://bioconductor.org/packages/edgeR)
    + [Workflow](https://plbaldoni.github.io/GibbsDTE-code)
    + [Paper](https://doi.org/10.1093/nargab/lqae151)

* catchSalmon/catchKallisto (within edgeR) ![](https://bioconductor.org/shields/years-in-bioc/edgeR.svg)
    + Estimation of mapping ambiguity overdispersion from transcript quantification of short read RNA-seq data. It unlocks uncertainty-free differential expression assessment at the transcript-level within edgeR.
    + [R/Bioconductor](https://bioconductor.org/packages/edgeR)
    + [Workflow](https://plbaldoni.github.io/TranscriptDE-code) and [User's Guide](https://bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf)
    + [Paper](https://doi.org/10.1093/nar/gkad1167)

* epigraHMM ![](https://bioconductor.org/shields/years-in-bioc/epigraHMM.svg)
    + A toolkit for the analysis of epigenomic datasets such as ChIP-seq, ATAC-seq, CUT&RUN, and CUT&Tag. It performs differential and consensus peak calling from multi-sample multi-condition datasets.
    + [R/Bioconductor](https://bioconductor.org/packages/epigraHMM)
    + [Vignette](https://bioconductor.org/packages/release/bioc/vignettes/epigraHMM/inst/doc/epigraHMM.html)
    + [Paper](https://onlinelibrary.wiley.com/doi/10.1111/biom.13477)
    
* ZIMHMM
    + A consensus peak caller for epigenomic datasets. It implements a fast hidden Markov model with mixed-effects zero-inflated negative binomial emissions using sample-specific random effects.
    + [GitHub](https://github.com/plbaldoni/ZIMHMM)
    + [Paper](https://onlinelibrary.wiley.com/doi/full/10.1111/biom.13083)
    
## Timeline

![](https://ghchart.rshah.org/4B9CD3/plbaldoni)

