<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">

    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap" rel="stylesheet">
    <style type="text/css">body {font-family: 'Roboto Slab', serif;}</style>

    <title>Pedro Baldoni</title>

    
          <script src="index_files/header-attrs-2.25/header-attrs.js"></script>
    
    
  </head>
  <body>

    
    <div class="container-fluid d-none d-lg-block d-xl-block">
      <div class="d-flex row flex-row vh-100">
        <div class="d-flex flex-column col-5 align-items-center text-center align-self-center">
          <div class="mx-auto text-center">
            <div class="p-3">
              <img src=avatar.png style="height:15rem" class="rounded">
            </div>
            <div class="p-2">
              <h1>Pedro Baldoni</h1>
            </div>
            <hr style="width:60%;"/>
            <div class="p-2">
              <div id="icon-list">
                <ul class="list-inline">
                                    <li class="list-inline-item">
                    <a href=https://scholar.google.com/citations?user=5lHSzXYAAAAJ>
                      <button type="button" class="btn btn-outline-dark mb-2">
                        Google Scholar
                      </button>
                    </a>
                  </li>
                                    <li class="list-inline-item">
                    <a href=https://twitter.com/plbaldoni>
                      <button type="button" class="btn btn-outline-dark mb-2">
                        Twitter
                      </button>
                    </a>
                  </li>
                                    <li class="list-inline-item">
                    <a href=https://github.com/plbaldoni>
                      <button type="button" class="btn btn-outline-dark mb-2">
                        GitHub
                      </button>
                    </a>
                  </li>
                                    <li class="list-inline-item">
                    <a href=mailto:baldoni.p@wehi.edu.au>
                      <button type="button" class="btn btn-outline-dark mb-2">
                        Email
                      </button>
                    </a>
                  </li>
                                  </ul>
              </div>
            </div>
          </div>
        </div>
        <div class="d-flex flex-column vh-100 border-left"></div>
        <div class="d-flex flex-column col px-5 align-items-center align-self-center mh-100" style="overflow-y: scroll;">
          <div class="my-5" id="content">
            <div id="bio" class="section level2">
            <h2>Bio</h2>
            <p>I am currently a Postdoctoral Fellow in the <a
            href="https://www.wehi.edu.au/people/gordon-smyth/486/gordon-smyth-lab-team">Smyth
            Lab</a> at the <a href="https://www.wehi.edu.au/">Walter and
            Eliza Hall Institute of Medical Research</a>. At the WEHI,
            most of my research has been focused on the analysis and
            interpretation of bulk and single-cell epigenomic,
            transcriptomic, and proteomics data. I enjoy developing new
            statistical methods and writing efficient bioinformatic
            tools that can help researchers make sense of their
            data.</p>
            <p>In 2020, I graduated with a PhD in <a
            href="https://sph.unc.edu/bios/biostatistics/">biostatistics</a>
            from the <a
            href="https://sph.unc.edu/bios/biostatistics/">University of
            North Carolina at Chapel Hill</a>. During my PhD, I
            developed statistical methods for the analysis of both bulk
            and single-cell epigenomic assays. At UNC, I was a graduate
            researcher in the <a
            href="https://sites.cscc.unc.edu/hchs/">Hispanic Community
            Health Study / Study of Latinos (HCHS/SOL)</a> and
            developed/applied statistical methods to analyze data from
            the HCHS/SOL under complex survey design settings.</p>
            </div>
            <div id="projects" class="section level2">
            <h2>Projects</h2>
            <ul>
            <li>catchSalmon/catchKallisto (within edgeR) <img
            src="https://bioconductor.org/shields/years-in-bioc/edgeR.svg" />
            <ul>
            <li>Estimation of mapping ambiguity overdispersion from
            transcript quantification of short read RNA-seq data. It
            unlocks uncertainty-free differential expression assessment
            at the transcript-level within edgeR.</li>
            <li><a
            href="https://bioconductor.org/packages/edgeR">R/Bioconductor</a></li>
            <li><a
            href="https://plbaldoni.github.io/TranscriptDE-code">Workflow</a>
            and <a
            href="https://bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf">User’s
            Guide</a></li>
            <li><a
            href="https://www.biorxiv.org/content/10.1101/2023.04.02.535231">Preprint</a></li>
            </ul></li>
            <li>epigraHMM <img
            src="https://bioconductor.org/shields/years-in-bioc/epigraHMM.svg" />
            <ul>
            <li>A toolkit for the analysis of epigenomic datasets such
            as ChIP-seq, ATAC-seq, CUT&amp;RUN, and CUT&amp;Tag. It
            performs differential and consensus peak calling from
            multi-sample multi-condition datasets.</li>
            <li><a
            href="https://bioconductor.org/packages/epigraHMM">R/Bioconductor</a></li>
            <li><a
            href="https://bioconductor.org/packages/release/bioc/vignettes/epigraHMM/inst/doc/epigraHMM.html">Vignette</a></li>
            <li><a
            href="https://onlinelibrary.wiley.com/doi/10.1111/biom.13477">Paper</a></li>
            </ul></li>
            <li>ZIMHMM
            <ul>
            <li>A consensus peak caller for epigenomic datasets. It
            implements a fast hidden Markov model with mixed-effects
            zero-inflated negative binomial emissions using
            sample-specific random effects.</li>
            <li><a
            href="https://github.com/plbaldoni/ZIMHMM">GitHub</a></li>
            <li><a
            href="https://onlinelibrary.wiley.com/doi/full/10.1111/biom.13083">Paper</a></li>
            </ul></li>
            </ul>
            </div>
            <div id="timeline" class="section level2">
            <h2>Timeline</h2>
            <p><img
            src="https://ghchart.rshah.org/4B9CD3/plbaldoni" /></p>
            </div>
          </div>
        </div>
      </div>
    </div>

    <div class="container-flex min-vh-100 d-block d-lg-none d-xl-none">
      <div class="d-flex flex-row justify-content-center">
        <div class="d-flex flex-col">
          <div class="mx-auto">
            <div class="p-3 mt-3 text-center">
              <img src=avatar.png style="height:15rem" class="rounded">
            </div>
            <div class="p-2 text-center">
              <h1>Pedro Baldoni</h1>
            </div>
            <hr style="width:60%;"/>

            <div class="row justify-content-center align-items-center">
              <div class="col-8 p-1 text-center">
                <div class="list-group">
                                    <a href=https://scholar.google.com/citations?user=5lHSzXYAAAAJ>
                    <button type="button" class="list-group-item list-group-item-action my-1 rounded">
                      Google Scholar
                    </button>
                  </a>
                                    <a href=https://twitter.com/plbaldoni>
                    <button type="button" class="list-group-item list-group-item-action my-1 rounded">
                      Twitter
                    </button>
                  </a>
                                    <a href=https://github.com/plbaldoni>
                    <button type="button" class="list-group-item list-group-item-action my-1 rounded">
                      GitHub
                    </button>
                  </a>
                                    <a href=mailto:baldoni.p@wehi.edu.au>
                    <button type="button" class="list-group-item list-group-item-action my-1 rounded">
                      Email
                    </button>
                  </a>
                                  </div>
              </div>
            </div>
            <div class="p pt-4 px-5">
              <div id="bio" class="section level2">
              <h2>Bio</h2>
              <p>I am currently a Postdoctoral Fellow in the <a
              href="https://www.wehi.edu.au/people/gordon-smyth/486/gordon-smyth-lab-team">Smyth
              Lab</a> at the <a href="https://www.wehi.edu.au/">Walter
              and Eliza Hall Institute of Medical Research</a>. At the
              WEHI, most of my research has been focused on the analysis
              and interpretation of bulk and single-cell epigenomic,
              transcriptomic, and proteomics data. I enjoy developing
              new statistical methods and writing efficient
              bioinformatic tools that can help researchers make sense
              of their data.</p>
              <p>In 2020, I graduated with a PhD in <a
              href="https://sph.unc.edu/bios/biostatistics/">biostatistics</a>
              from the <a
              href="https://sph.unc.edu/bios/biostatistics/">University
              of North Carolina at Chapel Hill</a>. During my PhD, I
              developed statistical methods for the analysis of both
              bulk and single-cell epigenomic assays. At UNC, I was a
              graduate researcher in the <a
              href="https://sites.cscc.unc.edu/hchs/">Hispanic Community
              Health Study / Study of Latinos (HCHS/SOL)</a> and
              developed/applied statistical methods to analyze data from
              the HCHS/SOL under complex survey design settings.</p>
              </div>
              <div id="projects" class="section level2">
              <h2>Projects</h2>
              <ul>
              <li>catchSalmon/catchKallisto (within edgeR) <img
              src="https://bioconductor.org/shields/years-in-bioc/edgeR.svg" />
              <ul>
              <li>Estimation of mapping ambiguity overdispersion from
              transcript quantification of short read RNA-seq data. It
              unlocks uncertainty-free differential expression
              assessment at the transcript-level within edgeR.</li>
              <li><a
              href="https://bioconductor.org/packages/edgeR">R/Bioconductor</a></li>
              <li><a
              href="https://plbaldoni.github.io/TranscriptDE-code">Workflow</a>
              and <a
              href="https://bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf">User’s
              Guide</a></li>
              <li><a
              href="https://www.biorxiv.org/content/10.1101/2023.04.02.535231">Preprint</a></li>
              </ul></li>
              <li>epigraHMM <img
              src="https://bioconductor.org/shields/years-in-bioc/epigraHMM.svg" />
              <ul>
              <li>A toolkit for the analysis of epigenomic datasets such
              as ChIP-seq, ATAC-seq, CUT&amp;RUN, and CUT&amp;Tag. It
              performs differential and consensus peak calling from
              multi-sample multi-condition datasets.</li>
              <li><a
              href="https://bioconductor.org/packages/epigraHMM">R/Bioconductor</a></li>
              <li><a
              href="https://bioconductor.org/packages/release/bioc/vignettes/epigraHMM/inst/doc/epigraHMM.html">Vignette</a></li>
              <li><a
              href="https://onlinelibrary.wiley.com/doi/10.1111/biom.13477">Paper</a></li>
              </ul></li>
              <li>ZIMHMM
              <ul>
              <li>A consensus peak caller for epigenomic datasets. It
              implements a fast hidden Markov model with mixed-effects
              zero-inflated negative binomial emissions using
              sample-specific random effects.</li>
              <li><a
              href="https://github.com/plbaldoni/ZIMHMM">GitHub</a></li>
              <li><a
              href="https://onlinelibrary.wiley.com/doi/full/10.1111/biom.13083">Paper</a></li>
              </ul></li>
              </ul>
              </div>
              <div id="timeline" class="section level2">
              <h2>Timeline</h2>
              <p><img
              src="https://ghchart.rshah.org/4B9CD3/plbaldoni" /></p>
              </div>
            </div>
        </div>
      </div>
    </div>

    <!-- Optional JavaScript -->
    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>

  
  </body>
</html>
